contributors/devel/sig-node/e2e-node-tests.md

```diff
@@ -202,6 +196,46 @@ For example, the [`ci-kubernetes-node-kubelet`](https://github.com/kubernetes/te
 make test-e2e-node REMOTE=true FOCUS="\[NodeConformance\]" SKIP="\[Flaky\]|\[Serial\]"
 ```
 
+See [The Spec Runner](http://onsi.github.io/ginkgo/#the-spec-runner) in the Grinkgo documentation to learn more about how FOCUS and SKIP work.
+
+## Run a subset of tests using image configuration files
+
+Image configuration files can further influence FOCUS and SKIP match test cases.
+
+For example:
+
+```sh
+--focus="\[NodeFeature:.+\]" --skip="\[Flaky\]|\[Serial\]"
+```
+
+runs all e2e tests labeled `"[NodeFeature:*]"` and will skip any tests labeled `"[Flaky]"` or `"[Serial]"`.
+
+Two example e2e tests that would match this expression are:
+  * https://github.com/kubernetes/kubernetes/blob/0e2220b4462130ae8a22ed657e8979f7844e22c1/test/e2e_node/security_context_test.go#L155
+  * https://github.com/kubernetes/kubernetes/blob/0e2220b4462130ae8a22ed657e8979f7844e22c1/test/e2e_node/security_context_test.go#L175
```

Comment:
I think this would go better in the previous section.

Reply:
Are you referring to the section titled `Run tests using a custom image configuration`?